### PR TITLE
Add Tasks.ListPaginated

### PR DESCRIPTION
**Pull request by shimonp21**

### CODE REVIEW

File: task.go [hunk 1]
```diff
@@ -1,5 +1,10 @@
 package onfleet
 
+type PaginatedTasks struct {
```
Comment (naszzz) [inline task.go:3]: for consistency with rest of file I suggest naming this `TasksPaginated` instead of `PaginatedTasks`

File: task.go [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+	LastID string `json:"lastId"`
```
Comment (naszzz) [inline task.go:4]: change LastID to LastId and add omitempty
refer to to TeamTasks struct in team.go which also includes LastId.

File: service/task/client.go [hunk 3]
```diff
@@ -55,7 +55,20 @@ func (c *Client) GetByShortId(taskShortId string) (onfleet.Task, error) {
 	return task, err
 }
 
-func (c *Client) ListTasks(teamId string) {
+// Reference https://docs.onfleet.com/reference/list-tasks
+func (c *Client) ListPaginated(params onfleet.TaskListQueryParams) (onfleet.PaginatedTasks, error) {
```
Comment (naszzz) [inline service/task/client.go:59]: I'd suggest naming this simply List as user can understand from return type that LastId is/can be present for cases where multiple pages are present.

**Review by naszzz — CHANGES_REQUESTED** [review]

**Comment by shimonp21** [comment]
@naszzz Thanks for the quick review! 
[BTW, after this PR is approved + merged, would also be happy for a new release so I can use it in cq-source-onfleet :) ]

**Review by naszzz — APPROVED** [review]
thanks : ) i'm going to create a new release now!

**Pull request merged**